cli/pack/rpm_header.go

```diff
@@ -231,13 +232,27 @@ func getFilesInfo(relPaths []string, dirPath string) (filesInfo, error) {
 		info.FileUserNames = append(info.FileUserNames, defaultFileUser)
 		info.FileGroupNames = append(info.FileGroupNames, defaultFileGroup)
 		info.FileLangs = append(info.FileLangs, defaultFileLang)
-		info.FileLinkTos = append(info.FileLinkTos, defaultFileLinkTo)
 
 		sysFileInfo, ok := fileInfo.Sys().(*syscall.Stat_t)
 		if !ok {
 			return info, fmt.Errorf("failed to get file info")
 		}
 
+		// If file is a symlink, it is needed to be the relative one.
+		if fileInfo.Mode().Type() == fs.ModeSymlink {
+			currentDir, err := os.Getwd()
+			if err != nil {
+				return filesInfo{}, err
+			}
+			os.Chdir(filepath.Dir(fullFilePath))
+			path, err := filepath.EvalSymlinks(filepath.Base(fullFilePath))
```

Comment:
Why don't you use ResolveSymlink from util package? It returns the same info but without making chdir.